light-system-programs/programs/verifier_program_zero/Cargo.toml

```diff
@@ -17,12 +17,12 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = { git = "https://github.com/coral-xyz/anchor", rev = "1c6f86e5f7793ce6adefe9cbfa11939647c509ce" }
-anchor-spl = { git = "https://github.com/coral-xyz/anchor", rev = "1c6f86e5f7793ce6adefe9cbfa11939647c509ce" }
+anchor-lang = "0.28.0"
+anchor-spl = "0.28.0"
 solana-security-txt = "1.1.0"
 merkle_tree_program = { path = "../merkle_tree_program", features = ["cpi"] }
 
 # Light Deps
-groth16-solana = "0.0.1"
+groth16-solana = { git = "https://github.com/Lightprotocol/groth16-solana", branch = "vadorovsky/fix-add-decompression" }
```

Comment:
what do you think about using main for the actual merge?
Then we don't need to change this when I merge the other groth16 solana changes.

Reply:
My branch in groth16-solana is based in yours and I already filed a PR https://github.com/Lightprotocol/groth16-solana/pull/2 to your branch, not to main.

I think we should first merge https://github.com/Lightprotocol/groth16-solana/pull/2 into your branch, then review and merge your PR https://github.com/Lightprotocol/groth16-solana/pull/1, then switch to main here and merge this PR. Do you agree?

Reply:
Fair, let's do that xD

Reply:
converted to draft so nobody merges this